robot/rover/comms_demo/arduino_demos/pong_bing/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ configure_file(${PROJECT_NAME}.ino ${PROJECT_NAME}.ino.cpp)
 add_executable(${PROJECT_NAME}_EXE ${PROJECT_NAME}.ino.cpp)
 target_compile_options(${PROJECT_NAME}_EXE PUBLIC ${ARM_GXX_FLAGS})
 target_include_directories(${PROJECT_NAME}_EXE PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/includes)
-target_include_directories(${PROJECT_NAME}_EXE PRIVATE /home/cedric/Arduino/libraries/Adafruit_Motor_Shield_V2_Library)
+target_include_directories(${PROJECT_NAME}_EXE PRIVATE /home/michael/Arduino/libraries/Adafruit_Motor_Shield_V2_Library)
```

Comment:
same here

Reply:
Whoops, you're right. Lemme change that

Reply:
Ok then please add it to .gitignore in a way that makes sense

Reply:
I'll approve, but please do it before merging

Reply:
Will do